src/Data/HashTable/ST/Basic.hs

```diff
@@ -608,6 +665,10 @@ deletedMarker = 1
 
 
 ------------------------------------------------------------------------------
+trueInt :: Int -> Bool
```

Comment:
We probably want `isTrue#` where it's available, yes -- but we can fall back to `tagToEnum#` since that's what `isTrue#` does :)